share/share_test.go

```diff
@@ -238,3 +238,15 @@ func TestShareToBytesAndFromBytes(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, shares, reconstructedShares)
 }
+
+func TestMarshalShare(t *testing.T) {
+	sh := RandShares(1)
```

Comment:
```suggestion
	sh := RandShares(1)[0]
```

So you don't need to keep `[0]` below.